src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,181 @@
+
+The window title is the application’s name if no workspace is opened and `<workspace name> — <application name>` if a workspace is opened.
+The application name can be adapted in `electron-app/package.json`:
+Open the file and adapt the value of property `theia.frontend.config.applicationName` to the desired name.
+
+#### Application Icons
+
+The application’s icons are located in `electron-app/resources/`.
+Just replace them with your own icons.
+Because each operating system handles icons differently, you need to replace them for all of them.
+They map as follows:
+
+- MacOS: icons.icns
+- Windows: icon.ico
+- Linux: icons subfolder
+
+### Customizing the welcome page
+
+Theia’s welcome page can be customized by binding a custom `WidgetFactory` for Theia’s `GettingStartedWidget`.
+This is done by the Theia Blueprint product in the theia-example-updater extension.
+The easiest way to customize the welcome page is to adapt the class `TheiaInstallerGettingStartedWidget` in `theia-example-updater/src/electron-browser/customization/theia-installer-getting-started-widget.tsx`.
+
+The widget is bound in `theia-example-updater/src/electron-browser/theia-updater-frontend-module.ts` like this:
+
+```typescript
+bind(TheiaInstallerGettingStartedWidget).toSelf();
+  bind(WidgetFactory).toDynamicValue(context => ({
+      id: GettingStartedWidget.ID,
+      createWidget: () => context.container.get<TheiaInstallerGettingStartedWidget>(TheiaInstallerGettingStartedWidget),
+  })).inSingletonScope();
+```
+
+To use another custom widget, remove this code and bind your widget correspondingly.
+
+### Customizing the about dialog
+
+Theia’s about dialog can be customized by binding a custom subclass of Theia’s `AboutDialog` class to `AboutDialog`.
+This is done by the Theia Blueprint product in the theia-example-updater extension.
+The easiest way to customize the About dialog is to adapt the class `TheiaInstallerAboutDialog` in `theia-example-updater/src/electron-browser/customization/theia-installer-about-dialog.tsx`.
+
+The widget is bound in `theia-example-updater/src/electron-browser/theia-updater-frontend-module.ts` like this:
+
+```typescript
+isBound(AboutDialog)
+  ? rebind(AboutDialog).to(TheiaInstallerAboutDialog).inSingletonScope()
+  : bind(AboutDialog).to(TheiaInstallerAboutDialog).inSingletonScope();
+```
+
+To use another custom dialog widget, remove this code, extend Theia’s AboutDialog class, and (re)bind it like above.
+
+### Customizing the installer
+
+The installers are created using [electron-builder](https://www.electron.build/).
+The corresponding configuration file is located at `electron-app/electron-builder.yml`.
+
+#### Installer File Base Name
+
+The installer files’ base names are defined by the `productName` property in `electron-app/electron-builder.yml`.
+
+#### Windows Installer
+
+As is typical for Windows applications, there is an installation wizard for the windows version of the Theia blueprint.
```

Comment:
I believe we mentioned it previously, but we should be consistent when naming `Theia Blueprint`.

```suggestion
As is typical for Windows applications, there is an installation wizard for the windows version of Theia Blueprint.
```